app/models/history.rb

```diff
@@ -39,12 +39,12 @@ class History < ApplicationRecord
 
   belongs_to :patient
 
-  after_create :touch_patient
+  # patient updated_at should not be updated if history was created by a system generated action
+  after_create(proc { |history| history.patient.touch unless system_generated_action? })
```

Comment:
what about `after_create Proc.new { |history| history.patient.touch unless system_generated_action? }`

Reply:
These two things seem inconsistent: `history.patient.touch` vs. `system_generated_action?` (should `system_generated_action?` be `history.system_generated_action?`?)

Reply:
that's what I had originally but rubocop was being extremely picky about how it wanted the syntax to be, as for the inconsistency, I just tried it out and it seems like any combination of it works, so I just changed it to:
`after_create(proc { patient.touch unless system_generated_action? })`

Reply:
Sorry additional nitpick that isn't a big deal: `system_generated_action` is a bit misleading given the download is not system generated, and there are technically other system generated actions that don't trigger this. Maybe like `system_report_action_or_download?` That's not great either I'm just spitballing

Reply:
updated!